integrations/kube-agent-updater/cmd/teleport-kube-agent-updater/constants.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+var teleportProdOCIPubKey = []byte(`-----BEGIN PUBLIC KEY-----
```

Comment:
Where does it come from?
How can we update it when time comes?

(missing license header)

Can we add that as a comment?

Reply:
I fetched this from the prod KMS, @tcsc asked at some point how we will share our signature key, so I suspect we'll advertise it somewhere in the docs at least.

I'll add a comment explaining where to find they key and what to do in case of rotation.

Reply:
@hugoShaka @tcsc Would it make sense to store this public key as a separate file somewhere in the repo and embed using `go:embed` so it can potentially be reused in multiple places without hardcoding/copy-pasting?

Reply:
Yes, but if we dynamically load it we must be very careful not to update the file in place. In case of rotation, we might want to add a new validator for the new key instead of just replacing the key.